apps/studio/src/features/editing-experience/components/LayoutSelection.tsx

```diff
@@ -0,0 +1,245 @@
+// Component for layout selection
+
+import React, { useState } from 'react'
```

Comment:
nit: unnecessary React import
```suggestion
import { useState } from 'react'
```